alpine/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV KONG_VERSION 1.4.0
 ENV KONG_SHA256 e31080487cfb8262a839c19d5e8fdb18b7686e9e4cc594803eaa565e0b34ff7a
 
 
-RUN adduser -Su 1337 kong \
+RUN adduser -S kong \
 	&& mkdir -p "/usr/local/kong" \
 	&& apk add --no-cache --virtual .build-deps curl wget tar ca-certificates \
```

Comment:
I know it's not a blocker for this PR nor it's relevant but can we remove packages like curl, wget, tzdata, libcap and su-exec for security purposes? I don't think these are necessary as tests were all green for the 8fb68f1 commit before: https://travis-ci.com/Kong/docker-kong/builds/129166652

I'm also going to close https://github.com/Kong/docker-kong/pull/296 since we're cherry-picking it here.

Reply:
Isn't tzdata needed for timezones to work correctly?